Applications/ConsoleReferencePublisher/Program.cs

```diff
@@ -424,9 +573,11 @@ private static PublishedDataSetDataType CreatePublishedDataSetSimple()
                         ValueRank = ValueRanks.Scalar
                     },
                 };
+
+            // set the ConfigurationVersion relative to kTimeOfConfiguration constant
             publishedDataSetSimple.DataSetMetaData.ConfigurationVersion = new ConfigurationVersionDataType() {
-                MinorVersion = 1,
-                MajorVersion = 1
+                MinorVersion = ConfigurationVersionUtils.CalculateVersionTime(kTimeOfConfiguration),
```

Comment:
e.g. such a default init could be part of a partial class in core, extending ConfigurationVersionDataType